src/ert/plugins/hook_implementations/forward_model_steps.py

```diff
@@ -73,7 +73,7 @@ def documentation() -> ForwardModelStepDocumentation | None:
         return ForwardModelStepDocumentation(
             category="utility.file_system",
             description="""
-The job copies the directory :code:`<FROM>` to the target :code:`<TO>`. If
+The the directory :code:`<FROM>` to the target :code:`<TO>`. If
```

Comment:
The the search and replace went a bit fast!

Reply:
Fixed.